### PR TITLE
Resolves has2k1/plotnine/issues/536.

### PR DESCRIPTION
**Pull request by hyiltiz**
Signed-off-by: Hörmetjan Yiltiz <hyiltiz@gmail.com>

### CODE REVIEW

**Comment by hyiltiz** [comment]
@has2k1 what can I do before this can be merged? I am afraid the patch may get obsolete if starts to lag behind `master` too much.

File: plotnine/tests/test_theme.py [hunk 1]
```diff
@@ -143,6 +143,11 @@ def test_theme_minimal(self):
 
         assert p + _theme == 'theme_minimal'
 
+    def test_theme_tufte(self):
+        p = self.g + labs(title='Theme Tufte') + theme_tufte()
+
+        assert p + _theme == 'theme_tufte'
```
Comment (has2k1) [inline plotnine/tests/test_theme.py:149]: You need to include a baseline image, as is the case for [all other themes](https://github.com/has2k1/plotnine/tree/f71d043c367927929fe0736ef6bc405767ff0e8d/plotnine/tests/baseline_images/test_theme).

**Comment by has2k1** [comment]
> @has2k1 what can I do before this can be merged? I am afraid the patch may get obsolete if starts to lag behind `master` too much.

Need to fix the test issues (plus include an image).

**Comment by hyiltiz** [comment]
I edited some test, and dug around the code base, and tried running `pytest` manually, but haven't quite figured out how to manually generate the `plotnine/tests/baseline_images/test_theme/theme_tufte.png`. It seems I would have to manually load one of the `.csv` data sets, and run a few lines of code to plot it. Do you have the code block that does it? If you take a look at the actual `theme_tufte` implementation, it is actually very minimal: in fact, it is very similar to `theme_minimal`. At this point, I've spent about twice the time I implemented `theme_tufte` in trying to figure out implementing unit tests, how to pass them and get it merged etc.

Could you help tell me what to do more specifically so this can be merged or do it yourself as you are probably more familiar with the code base? Reading `README.rst`, `how-to-release.rst` and `Makefile` didn't make it clear either. It would be helpful if there is a `Readme-dev.rst` file that would explain a bit more about the project internals, maintenance logistics, and whatever is necessary for someone like me to get my code merged.

**Comment by TyberiusPrime** [comment]
> ven't quite figured out how to manually generate the `plotnine/tests/baseline_images/test_theme/theme_tufte.png`.

What I do, is to run the test, verify the output image manually and copy it into the right (baseline_images) location.

**Comment by has2k1** [comment]
Thank you @hyiltiz. I have merge this in https://github.com/has2k1/plotnine/commit/36a8f49e96cf6bd9e0af27ea36dc838226967af2 and fixed the styling issues.

I will add the instructions on how to generate the test images. @TyberiusPrime is spot on.

**Comment by hyiltiz** [comment]
Thank you so much! I did run the tests, expecting them to generate the baseline images, but wasn't sure if I could take the first test image as the baseline (seems like that somewhat defeats the purpose of the test; it only guarantees later tests are the same as the first) so didn't do it myself.
Thanks for your help @has2k1 @TyberiusPrime 

Are you planning to cherry-pick/rebase merge manually, or merge via Github? I wonder how those would show up in my Github stats page.

**Comment by has2k1** [comment]
I rebased, squashed and merged already. [Here](https://github.com/has2k1/plotnine/commit/e349d692934dc5e3b28896458dfa960fe7f9ebf6), it shows you as the author and me as the committer.

**Comment by hyiltiz** [comment]
`Readme.md` cites `./doc/images/readme-image-5alt.png` which doesn't exist in the current master. To create it, simply run the code snippets in `Readme.md`, then instead of the final `theme_xkdc()`, use `theme_tufte()` instead.

**Comment by has2k1** [comment]
> `Readme.md` cites `./doc/images/readme-image-5alt.png` which doesn't exist in the current master. To create it, simply run the code snippets in `Readme.md`, then instead of the final `theme_xkdc()`, use `theme_tufte()` instead.

Thanks, I have generated it.

**Pull request closed without merging**